### PR TITLE
{vis}[GCCcore/6.4.0] SDL_image v1.2.12

### PR DESCRIPTION
**Pull request by JackPerdue**
(created using `eb --new-pr`)

### CODE REVIEW

**Review by boegel — DISMISSED** [review]

**Comment by migueldiascosta** [comment]
@JackPerdue, this PR is being closed for the following reason(s): closing and reopening to trigger tests.

File: easybuild/easyconfigs/l/libxml2/libxml2-2.9.8-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name = 'libxml2'
+version = '2.9.8'
```
[the remark this answers is not in the document]
Comment (akesandgren) [reply easybuild/easyconfigs/l/libxml2/libxml2-2.9.8-GCCcore-6.4.0.eb]: Done

File: easybuild/easyconfigs/s/SDL/SDL-1.2.15-GCCcore-6.4.0.eb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+easyblock = 'ConfigureMake'
+
+name = 'SDL'
+version = '1.2.15'
+
+homepage = 'http://www.libsdl.org/'
+
+description = """
+ SDL: Simple DirectMedia Layer, a cross-platform multimedia library
+"""
```
[the remark this answers is not in the document]
Comment (akesandgren) [reply easybuild/easyconfigs/s/SDL/SDL-1.2.15-GCCcore-6.4.0.eb]: Done

File: easybuild/easyconfigs/s/SDL/SDL-1.2.15-GCCcore-6.4.0.eb [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+toolchainopts = {'pic': True}
+
+# http://www.libsdl.org/release/SDL-1.2.15.tar.gz
```
[the remark this answers is not in the document]
Comment (akesandgren) [reply easybuild/easyconfigs/s/SDL/SDL-1.2.15-GCCcore-6.4.0.eb]: Done

File: easybuild/easyconfigs/s/SDL/SDL-1.2.15-GCCcore-6.4.0.eb [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+source_urls = ['http://www.libsdl.org/release/']
+sources = [SOURCE_TAR_GZ]
+patches = ['SDL-1.2.15-91ad7b43317a.patch']  # from: https://bugzilla.libsdl.org/show_bug.cgi?id=1769
```
[the remark this answers is not in the document]
Comment (akesandgren) [reply easybuild/easyconfigs/s/SDL/SDL-1.2.15-GCCcore-6.4.0.eb]: Done

File: easybuild/easyconfigs/s/SDL/SDL-1.2.15-GCCcore-6.4.0.eb [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    ('binutils', '2.28'),
+]
+
+dependencies = [
+    ('X11', '20180131'),
+]
+
+preconfigopts = """
+    ./autogen.sh  # else we get error: conflicting types for ‘_XData32’... even with patch above
+"""
```
[the remark this answers is not in the document]
Comment (akesandgren) [reply easybuild/easyconfigs/s/SDL/SDL-1.2.15-GCCcore-6.4.0.eb:33]: Done

File: easybuild/easyconfigs/s/SDL/SDL-1.2.15-GCCcore-6.4.0.eb [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+sanity_check_paths = {
+    'dirs': ['bin', 'include/SDL', 'lib/pkgconfig', 'share/man/man3'],
+    'files': ['bin/sdl-config', 'lib/libSDL.%s' % SHLIB_EXT],
+}
+
+moduleclass = 'lib'
+# EOF
```
[the remark this answers is not in the document]
Comment (akesandgren) [reply easybuild/easyconfigs/s/SDL/SDL-1.2.15-GCCcore-6.4.0.eb]: Done

File: easybuild/easyconfigs/s/SDL_image/SDL_image-1.2.12-GCCcore-6.4.0.eb [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+easyblock = 'ConfigureMake'
+
+name = 'SDL_image'
+version = '1.2.12'
+
+homepage = 'https://www.libsdl.org/projects/SDL_image/release-1.2.html'
+
+description = """
+ SDL_image is an image file loading library. 
+"""
```
[the remark this answers is not in the document]
Comment (akesandgren) [reply easybuild/easyconfigs/s/SDL_image/SDL_image-1.2.12-GCCcore-6.4.0.eb]: Done

File: easybuild/easyconfigs/s/SDL_image/SDL_image-1.2.12-GCCcore-6.4.0.eb [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+# https://www.libsdl.org/projects/SDL_image/release/SDL_image-1.2.12.tar.gz
```
[the remark this answers is not in the document]
Comment (akesandgren) [reply easybuild/easyconfigs/s/SDL_image/SDL_image-1.2.12-GCCcore-6.4.0.eb]: Done

File: easybuild/easyconfigs/s/SDL_image/SDL_image-1.2.12-GCCcore-6.4.0.eb [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+
+dependencies = [
+    ('libjpeg-turbo', '1.5.3'),
+    ('libpng', '1.6.34'),
+    ('LibTIFF', '4.0.9'),
+    ('SDL', '1.2.15'),
+]
+
+sanity_check_paths = {
+    'files': ['include/SDL/SDL_image.h', 'lib/libSDL_image.so',
```
[the remark this answers is not in the document]
Comment (akesandgren) [reply easybuild/easyconfigs/s/SDL_image/SDL_image-1.2.12-GCCcore-6.4.0.eb]: Done

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
b-an04.hpc2n.umu.se - Linux ubuntu 16.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/f7281233ea2888335798568c2fce7cc6 for a full test report.

**Review by akesandgren — APPROVED** [review]
LGTM

**Comment by akesandgren** [comment]
Going in, thanks @JackPerdue!

**Pull request merged**